### ISSUE TITLE
About the loss function type

### ISSUE THREAD

**Issue by FANnn100** [bisno/WDRO#12]
Thanks for the work. 
The loss functions mentioned in this paper are mainly the cross-entropy and MSE used in traditional regression and classification tasks. 
I am curious whether the implementation methods in the paper can be compatible with any other loss functions.

**Comment by bisno** [comment]
Thank you.

We have also tried **torch.nn.BCELoss** for multi-lable classification, and it also works.
